p2p/tinder/driver.go

```diff
@@ -0,0 +1,56 @@
+package tinder
```

Comment:
are p2p/tinder and p2p/ipfsutil mostly copied code?

Reply:
yes - I took from a project I found while looking for correct usages of libp2p (see point 2- Use berty project's service code) https://github.com/berty/berty